p2p/peer.go

```diff
@@ -307,7 +307,6 @@ func (p *Peer) pingLoop() {
 				p.protoErr <- err
 				return
 			}
-			ping.Reset(pingInterval)
```

Comment:
In this particular case, the Ticker is not equivalent, because we reset the timer after sending the message. Note the sending can take quite a bit here, if another large message is delivered when the timer fires. So I think we should leave this as is.

Reply:
Done